CLR/Libraries/Windows_Devices/windows_devices_stub.cpp

```diff
@@ -59,3 +59,31 @@ HRESULT Library_windows_devices_native_Windows_Devices_Gpio_GpioPin::SetDriveMod
     TINYCLR_SET_AND_LEAVE(CLR_E_NOT_SUPPORTED);
     TINYCLR_NOCLEANUP();
 }
+
+HRESULT Library_windows_devices_native_Windows_Devices_Spi_SpiBusInfo::get_ChipSelectLineCount___I4( CLR_RT_StackFrame& stack )
+{
+    TINYCLR_HEADER();
+    TINYCLR_SET_AND_LEAVE(CLR_E_NOT_SUPPORTED);
+    TINYCLR_NOCLEANUP();

```

Comment:
perfectly fine, consider implementing a helper method in the CLR_RT_StackFrame along the lines of:



HRESULT CLR_RT_StackFrame::NotImplementedStub()
{
    NATIVE_PROFILE_CLR_CORE();
    TINYCLR_HEADER();

    if(m_call.m_target->retVal != DATATYPE_VOID)
    {
        SetResult_I4( 0 );
    }

    TINYCLR_SET_AND_LEAVE(CLR_E_NOTIMPL);

    TINYCLR_NOCLEANUP();
}

it leaves the managed stack and managed object cleaner in which it sets the result to a nice round 0